terraform/day1/README.md

```diff
@@ -0,0 +1,41 @@
+![TechSlate](../global/images/ts.png)
+
+# Introduction 
+
+This page contains the links of all sub sections of day1 Terraform .
+
+## What is Terraform?
+
+- Terraform is an infrastructure as code tool that lets you build, change, and version cloud and on-prem resources safely and efficiently.
+
+- HashiCorp Terraform is an infrastructure as code tool that lets you define both cloud and on-prem resources in human-readable configuration files that you can version, reuse, and share. You can then use a consistent workflow to provision and manage all of your infrastructure throughout its lifecycle. Terraform can manage low-level components like compute, storage, and networking resources, as well as high-level components like DNS entries and SaaS features.
+
+## What is Resource Group?
+
+- A resource group is a logical container used in cloud computing environments to organize and manage related resources. It is a fundamental concept in cloud platforms such as Microsoft Azure, Amazon Web Services (AWS), and Google Cloud Platform (GCP). The purpose of a resource group is to simplify the management and organization of resources that belong to a specific application, project, or workload.
```

Comment:
as resource group is in azure so keep it as just azue dont include aws and gcp write defination more clearly